### DISCUSSION TITLE
Abort on error?

### DISCUSSION THREAD

**Discussion by hoerup** [renovatebot/renovate#17475] [Request Help] [answered]
I have observed that on a repo, renovate has discovered the following files

```
 INFO: Dependency extraction complete (repository=groupname/repository1, baseBranch=master)
       "stats": {
         "managers": {
           "docker-compose": {"fileCount": 4, "depCount": 2},
           "dockerfile": {"fileCount": 3, "depCount": 4},
           "npm": {"fileCount": 1, "depCount": 9},
           "nuget": {"fileCount": 3, "depCount": 11}
         },
         "total": {"fileCount": 11, "depCount": 26}
       }
```

it runs into a auth error with index.docker.io, but then it skips the other managers all together - even though it could probably create PR's for some updates in those ?

Im not sure if this is as designed or a bug - therefore the discussion instead

**Comment by rarkins** [comment, marked as answer]
Intentional behaviour for known public repos like Docker and GitHub which may "go down" sometimes. If the problems are temporary then it means you'd have PRs getting auto closed and needing recreating once they're back up. If the problems are permanent then add auth or disable the packages so that Renovate can work normally.
